### PR TITLE
Removing ChannelMessage.ReadWrite.All permission and add a note for $top query expectation

### PR DESCRIPTION
**Pull request by sumanac**

### CODE REVIEW

File: api-reference/v1.0/api/chatmessage-update.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ One of the following permissions is required to call this API. To learn more, in
 |:---------------------------------------|:--------------------------------------------|
 | Delegated (work or school account)     | ChannelMessage.ReadWrite, Group.ReadWrite.All** |
 | Delegated (personal Microsoft account) | Not supported. |
-| Application                            | ChannelMessage.UpdatePolicyViolation.All, ChannelMessage.ReadWrite.All, Group.ReadWrite.All** |
```
Comment (eddie-lee-msft) [inline api-reference/v1.0/api/chatmessage-update.md:31]: do you need to remove this in beta too?

Comment (sumanac) [reply api-reference/v1.0/api/chatmessage-update.md:31]: Yes. this permission does not exist in AAD. So, don't need it anywhere.

Comment (eddie-lee-msft) [reply api-reference/v1.0/api/chatmessage-update.md:31]: I know, but I think you missed removing it in the corresponding beta doc too:

https://learn.microsoft.com/en-us/graph/api/chatmessage-update?view=graph-rest-beta&tabs=http

Comment (sumanac) [reply api-reference/v1.0/api/chatmessage-update.md:31]: added

**Review by eddie-lee-msft — APPROVED** [review]

**Review by eddie-lee-msft — APPROVED** [review]

File: api-reference/beta/api/chatmessage-post.md [hunk 2]
```diff
@@ -1372,6 +1372,223 @@ Content-type: application/json
...
+            "thumbnailUrl": null,
+            "teamsAppId": null
+        }
+    ],
+    "mentions": [],
+    "reactions": []
+}
+
+```
+### Example 12: Send message with emojis
```
Comment (JarbasHorst) [inline api-reference/beta/api/chatmessage-post.md]: Should "api-reference/v1.0/api/chatmessage-post.md" also contain an example for emoji?

File: changelog/Microsoft.Teams.Core.json [hunk 3]
```diff
@@ -6203,6 +6203,58 @@
...
+      "SubArea": ""
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "31b9ae7d-a747-4642-9779-1bd60e3951b8",
+          "ApiChange": "Property",
+          "ChangedApiName": "content",
+          "ChangeType": "Update",
+          "Description": "Added new emoji support to message body **content** to the [itemBody](https://learn.microsoft.com/graph/api/resources/itembody?view=graph-rest-1.0) resource.",
```
Comment (JarbasHorst) [inline changelog/Microsoft.Teams.Core.json:6240]: I don't see in this PR any associated documentation update for this property. Please, update the docs accordingly.

Same for line 6214.

File: api-reference/beta/api/chatmessage-post.md [hunk 4]
```diff
@@ -1372,9 +1372,201 @@ Content-type: application/json
...
+            "teamsAppId": null
+        }
+    ],
+    "mentions": [],
+    "reactions": []
+}
+
+```
+
+### Example 12: Send a message with an emoji
```
Comment (JarbasHorst) [inline api-reference/beta/api/chatmessage-post.md:1490]: Should "api-reference/v1.0/api/chatmessage-post.md" also contain an example for emoji?

File: api-reference/beta/api/chatmessage-post.md [hunk 5]
```diff
@@ -1372,6 +1372,223 @@ Content-type: application/json
...
+    "webUrl": "https://teams.microsoft.com/l/message/19%3A81f49626414645c99469ee65a1a7e1a4%40thread.tacv2/1675104302171?groupId=5c884e2f-83f8-4cff-af8e-0177f260b9f8&tenantId=2432b57b-0abd-43db-aa7b-16eadd115d34&createdTime=1675104302171&parentMessageId=1675104302171",
+    "onBehalfOf": null,
+    "policyViolation": null,
+    "eventDetail": null,
+    "from": {
+        "application": null,
+        "device": null,
+        "user": {
+            "id": "28c10244-4bad-4fda-993c-f332faef94f0",
+            "displayName": "Sumana Chakrabarti",
```
[the remark this answers is not in the document]
Comment (sumanac) [reply api-reference/beta/api/chatmessage-post.md]: done.

File: api-reference/beta/api/chatmessage-post.md (hunk 4 above)
Comment (sumanac) [reply api-reference/beta/api/chatmessage-post.md:1490]: no we are not rolling out this for V1.0 yet

File: changelog/Microsoft.Teams.Core.json (hunk 3 above)
Comment (sumanac) [reply changelog/Microsoft.Teams.Core.json:6240]: This is not actually a property update. This is actually a value of a property where emojis are added. Please suggest what is the correct value for this changelog.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**